src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -192,6 +188,12 @@
       right: 0em; // aligning chevron to the right
     }
   }
+
+  .toctree-collapsible-header {
```

Comment:
Is this a good classname?

Reply:
I think "heading" is better than "header" here.  Also technically the primary sidebar nav is not really a toctree, so maybe `sidebar-primary-collapsible-heading`? Or something along those lines; tweak as needed to achieve similarity to related class names.